controllers/helpers/create-form-router.js

```diff
@@ -134,27 +134,25 @@ function createFormRouter({ router, formModel }) {
                 });
             }
         })
-        .post(function(req, res) {
+        .post(async function(req, res) {
             const formData = getFormSession(req);
             const successStep = formModel.getSuccessStep();
             const errorStep = formModel.getErrorStep();
 
             if (isEmpty(formData)) {
                 res.redirect(req.baseUrl);
             } else {
-                successStep
-                    .processor(formModel, formData)
-                    .then(() => {
-                        res.redirect(`${req.baseUrl}/success`);
-                    })
-                    .catch(err => {
-                        Raven.captureException(err);
-                        res.render('pages/apply/error', {
-                            form: formModel,
-                            stepConfig: errorStep,
-                            returnUrl: `${req.baseUrl}/review`
-                        });
+                try {
```

Comment:
Didn't actually change anything here in the end other than switch to async/await.